packages/command/src/sfdxCommand.ts

```diff
@@ -118,8 +120,8 @@ export abstract class SfdxCommand extends Command {
   // Set to true if this command MUST be run within a SFDX project.
   protected static requiresProject = false;
 
-  // Set to true if this command is deprecated.
-  // protected static deprecated = false;
+  // Set if this command is deprecated.
+  protected static deprecated?: Deprecation;
```

Comment:
I'm not 100% sure this is right since the original approach seemed to use a boolean value... But I couldn't figure out how to support a version number otherwise.  Input welcome.